pkg/controllers/provisioning/provisioner.go

```diff
@@ -268,7 +259,7 @@ func (p *Provisioner) NewScheduler(ctx context.Context, pods []*v1.Pod, stateNod
 }
 
 func (p *Provisioner) Schedule(ctx context.Context) ([]*scheduler.Machine, []*scheduler.ExistingNode, error) {
-	defer metrics.Measure(schedulingDuration.WithLabelValues(injection.GetNamespacedName(ctx).Name))()
+	defer metrics.Measure(schedulingDuration)()
```

Comment:
Why don't we need to inject the namespaced name anymore?

Reply:
This didn't make any sense to begin with since the scheduling duration uses multiple provisioners within its scheduling loop.